src/libstd/keyword_docs.rs

```diff
@@ -29,9 +29,14 @@
 /// [`crate`]: keyword.crate.html
 mod as_keyword { }
 
+#[doc(keyword = "break")]
+//
+/// Exit early from a loop.
```

Comment:
Not true with https://github.com/rust-lang/rust/issues/48594 but it's unstable.

Reply:
I'm not sure to understand your @Centril

Reply:
@GuillaumeGomez That this is not a full description of what `break` does.

Reply:
Alright, since we are going for "mostly right but not full story", let's call this resolved.